ManageAxes.js

```diff
@@ -161,3 +170,79 @@ ManageAxes.resetAxes = async function()
 
     await WSM.APISetLocalCoordinateSystem(nHistoryID, defaultLCS);
 }
+
+// moves all geometry in the current editing history to the world origin, then applies the reverse transform to the group
+ManageAxes.reOrigin = async () => {
+    // get the current editing group instance path
+    const editingPath = await FormIt.GroupEdit.GetInContextEditingPath();
+    const editingHistoryId = await FormIt.GroupEdit.GetEditingHistoryID();
+
+    // this action won't be valid for the main history
+    if (editingHistoryId === 0) {
```

Comment:
I forgot if we have an api to get the id of the main history. If so, you should use it. Otherwise this is ok